notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -1811,7 +1817,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.8.6"
```

Comment:
I think your local copy of JupyterLab isn't the one that `env.sh` creates. That script installs Python 3.7.

Reply:
Oops, must have been a diff env. I'll fix it up and re-run